client/packages/system/src/Encounter/utils.ts

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from 'react';
+import {
+  DateUtils,
+  LocaleKey,
+  TypedTFunction,
+  useTranslation,
+} from '@common/intl';
+import { EncounterRowFragment } from './api';
+import { EncounterNodeStatus } from '@common/types';
+
+const effectiveStatus = (
```

Comment:
Just a question, what is the benefit of adding this field to the fragment vs just mapping in columnList ? It seems simpler in terms of types and less indirection i.e.

in `useEncounterListColumns`

```
  columnList.push({
    key: 'effectiveStatus',
    label: 'label.status',
    sortable: false,
    align: ColumnAlign.Right,
    width: 175,
    accessor: ({ rowData }) => rowData?.status,
    formatter: status => effectiveStatus(status, t) // or useEffectiveStatus without
  });
```

Reply:
a question for @clemens-msupply I'm afraid - this is just a move of his change. I don't think that the fragment version is more efficient, if we wanted that we should send the status from the api and allow react query to cache the results.

I would have done the mapping in columnList, as you suggest - in this case I left the field rather than refactor when Clemens isn't here. I should have probably pointed it out in the review which introduced the change.